util/util.go

```diff
@@ -470,6 +470,9 @@ func (k KubeAwareAPIVersions) Less(i, j int) bool {
 // ClusterToObjectsMapper returns a mapper function that gets a cluster and lists all objects for the object passed in
 // and returns a list of requests.
 // NB: The objects are required to have `clusterv1.ClusterNameLabel` applied.
+// Deprecated: This function is deprecated and will be removed in a future release, use ClusterToTypedObjectsMapper instead.
```

Comment:
So that this is correctly marked as Deprecated:
```suggestion
//
// Deprecated: This function is deprecated and will be removed in a future release, use ClusterToTypedObjectsMapper instead.
```

Reply:
Damn. I thought about this but forgot to do it :D 

Thx!